packages/espressocash_app/lib/features/swap/src/create_swap/components/token_dropdown.dart

```diff
@@ -44,6 +44,7 @@ class TokenDropDown extends StatelessWidget {
         dropdownColor: Colors.transparent,
         icon: const SizedBox.shrink(),
         underline: const SizedBox.shrink(),
+        borderRadius: const BorderRadius.all(Radius.circular(30)),
```

Comment:
No, the element has `StadiumBorder`, hardcoding radius here is error-prone.

Reply:
As discussed, we will create a custom widget to achieve this. 

It will be done on a separate PR (https://github.com/espresso-cash/espresso-cash-public/pull/852)